developer-docs-site/docs/tutorials/first-move-module-cli.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Your First Move Module using the CLI"
+slug: "first-move-module-cli"
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Your First Move Module
+
+This tutorial details how to compile, test, publish and interact with Move Modules on the Aptos Blockchain. The steps are:
+
+1. [Install the CLI from Git][install_cli]
+2. Compile and test a Move module
+3. Publish a Move module to the Aptos Blockchain
+4. Interact with a Move Module
+5. Understand the code
+
+
+## Prepare the CLI environment
+
+After installing the CLI from Git, prepare your environment for interacting with Aptos by creating and funding an account. Begin by starting a new terminal.
+
+1. Initialize a new local account: `aptos init`. This will output:
+```
+Enter your rest endpoint [Current: None | No input: https://fullnode.devnet.aptoslabs.com/v1]
+
+No rest url given, using https://fullnode.devnet.aptoslabs.com/v1...
+Enter your faucet endpoint [Current: None | No input: https://faucet.devnet.aptoslabs.com | 'skip' to not use a faucet]
+
+No faucet url given, using https://faucet.devnet.aptoslabs.com...
+Enter your private key as a hex literal (0x...) [Current: None | No input: Generate new key (or keep one if present)]
+
+No key given, generating key...
+Account a345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a doesn't exist, creating it and funding it with 10000 coins
+Aptos is now set up for account a345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a!  Run `aptos help` for more information about commands
+{
+  "Result": "Success"
+}
+```
+2. Now fund this account: `aptos account fund-with-faucet --account a345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a`
+```
+{
+  "Result": "Added 10000 coins to account a345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a"
+}
+```
+
+## Compile and test the module
+
+There are many example Move modules in the [aptos-core/aptos-move/move-examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples) directory.
+
+Load a terminal and change directories into the `hello_blockchain` directory: `cd aptos-core/aptos-move/move-examples`.
+
+To build the module run: `aptos move build --named-addresses hello_blockchain=0xa345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a`.
+To test the module run: `aptos move test --named-addresses hello_blockchain=0xa345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a`.
```

Comment:
Should you use the alias from the init step?  E.g --named-addresses hello_blockchain=default

Reply:
I'll land and leave it to you folks to adjust according to what you know.